lib/streamaggr/streamaggr.go

```diff
@@ -802,7 +802,7 @@ func (a *aggregator) Push(tss []prompbmarshal.TimeSeries, matchIdxs []byte) {
 	defer putPushCtx(ctx)
 
 	samples := ctx.samples
-	buf := ctx.buf
+	buf := ctx.buf[:0]
```

Comment:
it's already done here 
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/bff83230631437988939d5891e3f8a94e9ba9c40/lib/streamaggr/streamaggr.go#L928